book/src/runtime.md

```diff
@@ -58,10 +66,13 @@ pages.
 The interface is best described by the `Instruction::userdata` that the user
 encodes.
 
-* `CreateAccount` - This allows the user to create and assign an account to a
-  Program.
+* `CreateAccount` - This allows the user to create an account with allocated
+memory and assign it to a Program.
+
 * `Assign` - allows the user to assign an existing account to a program.
+
 * `Move`  - moves tokens between accounts.
+
 * `Spawn` - spawns a new program from an account.
```

Comment:
Might want to expand on what this means.  Aka, marks the account as executable, that its userdata contains the programs bits that are executed, and that it contains a reference to the loader that will execute it.